packages/@react-spectrum/menu/docs/Menu.mdx

```diff
@@ -295,19 +295,23 @@ within the Menu upon opening.
 
 ### Disabled
 
+To disable Menu items, pass an array of keys into the `disabledKeys` prop.
+If an item has both `key` and `id` attributes, the `key` attribute should be used for disabling.
```

Comment:
I think we can make this a bit more general. Each item has a single key, which may be derived from a number of places including the `key` prop on the `<Item>`, or the `key` or `id` property on the object. This is not specific to the disabled state: selection also uses the same derived keys. Perhaps we need to clarify this in the content section above? If we do that, we could probably remove this sentence here since you link to it below.

Reply:
@devongovett Does that mean you don't actually need to create an items object in order to use the disabled functionality? I was a little confused by that honestly. I had everything complete, then when it came to disabling items the docs appeared to make it clear that I had to change the entire structure from what the rest of the documentation told me to do, and use an object rather than an explicit <Item> component for each item. Maybe we could have the example here match the rest of the documentation if that's the case. Let me know if that didn't make sense :)

Reply:
Sorry for the delay on this Celeste. The example is showing a dynamic use of items as explained here - https://react-spectrum.adobe.com/react-spectrum/Menu.html#dynamic-items
In the docs we show either the dynamic use case or the static use case just to mix them up. Maybe that was the wrong choice. I would be happy if you changed this example to use the static API (to match the other examples) and we can see how it goes from there.